### PR TITLE
WS-1165 add org logo to position serializer

### PR DESCRIPTION
**Pull request by rgraber**

### CODE REVIEW

File: course_discovery/apps/api/tests/test_serializers.py [hunk 1]
```diff
@@ -1770,6 +1771,7 @@ def test_position_with_no_org(self):
             'organization_id': None,
             'organization_override': None,
             'organization_marketing_url': None,
+            'organization_logo_image_url': None,
```
Comment (mikix) [inline course_discovery/apps/api/tests/test_serializers.py:1774]: You ought to have a test that actually tests the new code path too.

File: course_discovery/apps/api/serializers.py [hunk 2]
```diff
@@ -309,7 +310,7 @@ class Meta:
         model = Position
         fields = (
             'title', 'organization_name', 'organization', 'organization_id', 'organization_override',
-            'organization_marketing_url', 'organization_uuid',
+            'organization_marketing_url', 'organization_uuid', 'organization_logo_image_url'
```
Comment (mikix) [inline course_discovery/apps/api/serializers.py:313]: Feels like we should have started this serializer with a nested serializer. Ah well.

**Review by mikix — APPROVED** [review]
There's a missing test, but approving now to not be a blocker.

File: course_discovery/apps/api/tests/test_serializers.py (hunk 1 above)
Comment (rgraber) [reply course_discovery/apps/api/tests/test_serializers.py:1774]: Not sure I understand. Isn't that covered in the previous test?

Comment (mikix) [reply course_discovery/apps/api/tests/test_serializers.py:1774]: Yeah, looks like it! Missed that line, when I was reviewing.

**Review by albemarle — APPROVED** [review]

**Pull request merged**